client/verta/tests/unit_tests/conftest.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+
+"""Pytest fixtures for use in client unit tests."""
+
+import os
+from unittest.mock import patch
+
+import pytest
+
+from verta._internal_utils._utils import Connection, Configuration
+from verta.credentials import EmailCredentials
+
+
+@pytest.fixture(scope="session")
```

Comment:
I scoped these to the entire test session (instead of the default per-function) because it didn't seem necessary to recreate these objects for every single test.

Reply:
Makes sense to me.  We should be able to recycle a single fixture here.  Good call